admin_manual/release_notes.rst

```diff
@@ -694,3 +734,4 @@ or PostgreSQL) to operate correctly.
 .. _syncing user backends: configuration/server/occ_command.html#syncing-user-accounts
 .. _the new marketplace: https://marketplace.owncloud.com
 .. _Open Build Service: https://download.owncloud.org/download/repositories/10.0/owncloud/
+.. _jjjkjkj: https://github.com/owncloud/core/blob/stable10/CHANGELOG.md
```

Comment:
jjjkjkj ? are you using vim ?

Reply:
Oh lol, that's by mistake. First I tried but then I decided against adding all the ticket links there. Will remove, thanks.

Reply:
Not using vim btw ^_^ I'm nano noob :+1:

Reply:
@pmaier1 forgot to remove ?

Reply:
removed ^^